src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderNoPackage3.java

```diff
@@ -14,19 +14,19 @@
 
 import static java.awt.Button.ABORT;
 import static java.io.File.createTempFile;
-import java.util.Map; // violation
+import java.util.Map; // violation '.*Map' should be separated from previous import group by one line.'
 import java.util.Map.Entry;
 import java.util.NoSuchElementException;
 
 import com.google.common.annotations.Beta;
 import com.google.common.collect.HashMultimap;
 
 
-import org.apache.*; // violation
+import org.apache.*; // violation ''org.apache.*' should be separated from previous import group by one line.'
```

Comment:
Let's drop import value from message.
Apply to others.

Reply:
done